### PR TITLE
add BootOrder key to Intel mockup

### PR DESCRIPTION
**Pull request by jacobsalmela**
## Summary and Scope

Adds a missing `BootOrder` key found on Intel nodes. 

## Issues and Related PRs

* Relates to MTL-2229
* needed for https://github.com/Cray-HPE/gru/pull/20

## Testing

Local

### Test description:

Built the container, queried the endpoint, validated the data was there.

## Risks and Mitigations

Low

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-1.5.0
+1.5.1
```
Comment (mjendrysik-hpe) [inline .version]: I just merged my changes, this will need to go to 1.5.3

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ Fixed - for any bug fixes
 Security - in case of vulnerabilities
 -->
 
+## [1.5.] - 2023-11-02
```
Comment (mjendrysik-hpe) [inline CHANGELOG.md]: Missing patch version, which is OK since you will need 1.5.3

**Review by mjendrysik-hpe — CHANGES_REQUESTED** [review]

**Comment by mjendrysik-hpe** [comment]
Sorry, I forgot to merge my changes yesterday. So you will need to pull a new master down.

File: .version (hunk 1 above)
Comment (jacobsalmela) [reply .version]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (jacobsalmela) [reply CHANGELOG.md]: Fixed

**Review by mjendrysik-hpe — APPROVED** [review]

**Pull request merged**